src/fastlane/release_actions/lib/fastlane/plugin/release_actions/helper/commit/commit.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'footer'
+
```

Comment:
The paths here are quite long --

```
src/fastlane/release_actions/lib/fastlane/plugin/release_actions/helper/commit/commit.rb
```

Could that just be,
```
src/fastlane/release_actions/commit.rb
```
?

Or maybe there is another way to reduce the nesting? `commit.rb` is 8 levels down from `src` in the directory tree. Let's assume the directory tree were well-balanced, with 7 children at each node. Then, there would be ~1,048,575 files sibling to `commit.rb` (leaves in the tree). However, this commit has ~tens of files. Now, certainly there are other reasons to organize directories than to achieve an optimally balanced n-ary tree. But, it does seem there's an opportunity to optimize.

(Same comment about the PR's file layout, more generally.)

Reply:
That's how Fastlane prescribes organizing the helpers. But I don't think it's strongly enforced

Reply:
I hate the paths personally, but don’t think it matters much practically. Usage of this will be through lanes and these details won’t be directly exposed.

Reply:
Ah, another thing is that we're nesting a bit to allow later expansion. For example, other fastlane plugins can go in `src/fastlane`. When I'm working on this, I start at `src/fastlane/release_actions` as that's the root of the project, effectively, in this mono-repo-type thing.